sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -171,22 +176,44 @@ export class ApplicationStudentsController extends BaseController {
     @Param("applicationId", ParseIntPipe) applicationId: number,
     @UserToken() studentToken: StudentUserToken,
   ): Promise<void> {
+    const isFulltimeAllowed = this.configService.isFulltimeAllowed;
     const programYear = await this.programYearService.getActiveProgramYear(
       payload.programYearId,
     );
+    if (
+      ![OfferingIntensity.fullTime, OfferingIntensity.partTime].includes(
+        payload.data.howWillYouBeAttendingTheProgram,
+      )
+    ) {
+      throw new UnprocessableEntityException(
```

Comment:
Can we change this to a `BadRequestException`. IMO it should be the same that the Nestjs validator would throw.
Besides that, can you please add the below comment.
"The check to validate the values for howWillYouBeAttendingTheProgram can be removed once the toggle for IS_FULL_TIME_ALLOWED is no longer needed and the types are hard-coded again in the form.io definition using the onlyAvailableItems as true.".

Reply:
Changed it and added the comment.